MindFlayer.js

```diff
@@ -158,6 +271,7 @@
             Hooks.call('MindFlayerInit', this)
             this.setDefaultTokens()
             this._initializeWebsocket()
+            this._initializeAmbilight()
             this._overrideGetBorderColorOnTokens()
             this._overrideHandleKeysOnKeyboardToHideUI()
             this._overrideCameraPanForTokenMovement()
```

Comment:
I think we are at a point where we should start moving code into submodules to maintain readability. Like
- modules/settings
- modules/websocket
- modules/tokencontrol
- modules/hideui
- modules/camera
- modules/ambilight

Or something like that

Reply:
Good point, I have created a new Issue #13